Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/SpeechInputHandler.cs

```diff
@@ -62,7 +62,7 @@ protected virtual void Start()
             for (int index = 0; index < keywordCount; index++)
             {
                 KeywordAndResponse keywordAndResponse = Keywords[index];
-                string keyword = keywordAndResponse.Keyword.ToLower();
+                string keyword = keywordAndResponse.Keyword.ToLower().TrimEnd();
```

Comment:
What is the behavior if there is whitespace at the beginning of a keyword? Should we trim both ends?